### DISCUSSION TITLE
Issue Upgrading Unsupported Mac

### DISCUSSION THREAD

**Discussion by OCUser26** [community/community#75058] [New to GitHub] [answered]
I recently updated my unsupported IMac from Catalina to Big Sur using OpenCore Legacy Patcher without any issues. I have attempted to do the same with my MacBook Air but I am having an issue. The MacBook Air will not boot up (on its own) unless the external drive I used for the installer is connected. If I attempt a reboot without the external drive I get a grey screen with circle and diagonal line. When I connect the drive and restart, I initially get the screen with “Install Big Sur OS” and “untitled” (my mac’s internal drive). After a couple of seconds, the boot process selects the internal disk and boots up into Big Sur without any problems.

Some details:
- MacBook Air 13 inch mid-2011
- 1.7 GHz Dual Core Intel Core 5
- 4 GB 1333 MHz DDR 3
- Intel HD Graphics 3000 384 MB
- Startup Disk: untitled
- upgrading from High Sierra to Big Sur

Again - if I boot up with the external drive connected, everything seems to work fine (not the most speedy computer experience but it works). 

Any idea what I need to do?

Thanks

**Comment by resslr** [comment]
Head to Recovery mode, click unlock disk and set your boot disk, and delete any "Install OSName" apps as that can confuse it, make sure your mac is up to date using the official updater in settings.

**Reply by OCUser26** [reply]
No luck.  I verified the disk is not locked.  I have selected the internal disk as the Startup Disk.  And no "InstallOSName" app is around to confuse the boot up process.  I have un first aid on all devices of the internal drive, through Disk Utility.  The only way to get it to boot up is to connect the external drive and select EFI Boot.

**Reply by resslr** [reply]
Hm, unfortunately I can't help you further as I really don't know, sorry man.

**Reply by OCUser26** [reply]
Appreciate the support nonetheless.  May have found my answer on Open Core Legacy Patcher site....

"**Cannot boot macOS without the USB**
By default, the OpenCore Patcher won't install OpenCore onto the internal drive itself during installs.

After installing macOS, OpenCore Legacy Patcher should automatically prompt you to install OpenCore onto the internal drive. However, if it doesn't show the prompt, you'll need to either [manually transfer (opens new window)](https://dortania.github.io/OpenCore-Post-Install/universal/oc2hdd.html)OpenCore to the internal drive's EFI or Build and Install again and select your internal drive.

Reminder that once this is done, you'll need to select OpenCore in the boot picker again for your hardware to remember this entry and auto boot from then on."

**Reply by resslr** [reply, marked as answer]
Alright! Idk if my response still counts as an "Answer" but if you feel it does feel free to click Answer Ty! Also glad you were able to solve it!

**Comment by resslr** [comment]
Ty
